samples/mesh/main.c

```diff
@@ -213,31 +214,26 @@ void main(void)
         pb_push(p++, NV20_TCL_PRIMITIVE_3D_VP_UPLOAD_CONST_X, 4);
         memcpy(p, constants_0, 4*4); p+=4;
 
+        p = pb_end(p);
```

Comment:
Nu-uh. :(

Reply:
What is this comment supposed to signal?

Reply:
That I'm an idiot and that pb_end no longer returns a value.